src/app/public/modules/flyout/flyout-instance.ts

```diff
@@ -88,6 +120,9 @@ export class SkyFlyoutInstance<T> implements OnDestroy {
     this._iteratorNextButtonClick.complete();
   }
 
+  /**
+   * Closes the flyout instance.
```

Comment:
In the table at https://developer.blackbaud.com/skyux/components/flyout#skyflyoutinstance-properties, we also say this emits the `closed` event. Just making sure we are deliberately removing this from the docs.

Reply:
I changed it back.